File: Python/FilmTayang.py
import sys, os
sys.path.insert(0, os.path.abspath(os.path.dirname(__file__)))
from FilmBioskop import FilmBioskop


class FilmTayang(FilmBioskop):
    def __init__(self, id=0, judul="", genre="", durasi=0, studio="",
                 klasifikasiUsia="", hargaTiket=0, jamTayang="", statusTayang=""):
        super().__init__(id, judul, genre, durasi, studio, klasifikasiUsia)
        self.__hargaTiket = hargaTiket
        self.__jamTayang = jamTayang
        self.__statusTayang = statusTayang

    # Getter
    def getHargaTiket(self):
        return self.__hargaTiket

    def getJamTayang(self):
        return self.__jamTayang

    def getStatusTayang(self):
        return self.__statusTayang

    # Setter
    def setHargaTiket(self, hargaTiket):
        self.__hargaTiket = hargaTiket

    def setJamTayang(self, jamTayang):
        self.__jamTayang = jamTayang

    def setStatusTayang(self, statusTayang):
        self.__statusTayang = statusTayang

File: PHP/FilmTayang.php
<?php

require_once "FilmBioskop.php";

class FilmTayang extends FilmBioskop {
    private int $hargaTiket;
    private string $jamTayang;
    private string $statusTayang;

    // Constructor default / berparameter
    public function __construct(
        int $id = 0,
        string $judul = "",
        string $genre = "",
        int $durasi = 0,
        string $studio = "",
        string $klasifikasiUsia = "",
        int $hargaTiket = 0,
        string $jamTayang = "",
        string $statusTayang = "",
        string $gambar = ""
    ) {
        parent::__construct($id, $judul, $genre, $durasi, $studio, $klasifikasiUsia, $gambar);
        $this->hargaTiket   = $hargaTiket;
        $this->jamTayang    = $jamTayang;
        $this->statusTayang = $statusTayang;
    }

    // Getter
    public function getHargaTiket(): int       { return $this->hargaTiket; }
    public function getJamTayang(): string     { return $this->jamTayang; }
    public function getStatusTayang(): string  { return $this->statusTayang; }

    // Setter
    public function setHargaTiket(int $hargaTiket): void        { $this->hargaTiket = $hargaTiket; }
    public function setJamTayang(string $jamTayang): void       { $this->jamTayang = $jamTayang; }
    public function setStatusTayang(string $statusTayang): void { $this->statusTayang = $statusTayang; }
}
